src/lib/queries.js

```diff
@@ -1,10 +1,13 @@
-import { Q } from 'cozy-client'
+import { Q, fetchPolicies } from 'cozy-client'
+
+const ONE_HOUR = 60 * 60 * 1000
```

Comment:
if we use realtime on io.cozy.contact, we can rise the fetchpolicies significantly

Reply:
It depends of the app using it so I prefer to let it as is.